src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/ParameterNameCheck.java

```diff
@@ -211,11 +212,11 @@ public void setIgnoreOverridden(boolean ignoreOverridden) {
     /**
      * Setter to access modifiers of methods where parameters are checked.
      *
-     * @param accessModifiers access modifiers of methods which should be checked.
+     * @param accessModifierOptions access modifiers of methods which should be checked.
      */
-    public void setAccessModifiers(AccessModifier... accessModifiers) {
-        this.accessModifiers =
-            Arrays.copyOf(accessModifiers, accessModifiers.length);
+    public void setAccessModifierOptions(AccessModifierOption... accessModifierOptions) {
```

Comment:
Property name is not expected to change. This will break configurations.

Reply:
Done